pkg/disttask/framework/storage/task_table.go

```diff
@@ -730,3 +730,19 @@ func (mgr *TaskManager) GetSubtasksWithHistory(ctx context.Context, taskID int64
 	}
 	return subtasks, nil
 }
+
+// GetAllSubtasks gets all subtasks.
```

Comment:
```suggestion
// GetAllSubtasks gets all subtasks with basic columns.
```

Reply:
please move this comment to the interface definition